### ISSUE TITLE
Add background mode back

### ISSUE THREAD

**Issue by frederickfogerty** [imgix/react-imgix#160]
**Before you submit:**

- [x] Please search through the existing issues (both open AND closed) to see if your feature has already been discussed. Github issue search can be used for this: https://github.com/imgix/react-imgix/issues?utf8=%E2%9C%93&q=is%3Aissue
- [x] Please take a moment to find out whether your idea fits with the scope and aims of the project

**Is your feature request related to a problem? Please describe.**
I would like to set images as the background image of components such as `div` so I can place other items within this element.

**Describe the solution you'd like**
```jsx
<Imgix type="bg" src="..."><Heading>Blog Header</Heading></Imgix>
```

We could make this work by requiring `width` and `height` to be specified, as determining this ourselves is too error prone.

**Comment by Sandstedt** [comment]
This is essential for our project. So we need to use an old version of react-imgix currently. Could use `object-fit` in newer projects, but when we need to support older browsers, it's not possible.

**Comment by frederickfogerty** [comment]
Due to the community desire to have this reimplemented, this is happening! I'll post any updates here when I have them 👍

**Comment by frederickfogerty** [comment]
We're getting very close now! Track the progress over in #236.

![image](https://user-images.githubusercontent.com/615334/49589019-9108fb00-f968-11e8-9661-c71d969a15af.png)

**Comment by adamraider** [comment]
I pulled this down and all tests pass, maybe someone can kick off a rebuild on Travis?

**Comment by frederickfogerty** [comment]
🎉 Released in `8.5.0`. Thanks to everyone for their support and @adamraider for contributing to the PR!
